core/modules/block/block.info

```diff
@@ -5,3 +5,8 @@ version = BACKDROP_VERSION
 type = module
 backdrop = 1.x
 configure = admin/structure/block
+
+; Added by Backdrop CMS packaging script on 2017-09-15
```

Comment:
Delete "Added by..."

Also, there seem to a few files that show as having changed file permissions. We should try to avoid that.